desktop/cmp/rest/data/RestStore.ts

```diff
@@ -68,17 +81,17 @@ export class RestStore extends UrlStore {
         return resp;
     }
 
-    async addRecordAsync(rec) {
+    async addRecordAsync(rec: {id: string, data: PlainObject}) {
```

Comment:
Should we type this as `UnsavedStoreRecordConfig` or similar?

Reply:
I know the name is terrible lol

Reply:
yea, could be useful as a local type-- wondering if we need to export

Reply:
punting on this for now